src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Machine - excavator started working");
```

Comment:
The message is not informative enough. It should indicate that it is an Excavator that is working, not just any 'Machine - excavator'. Consider changing it to something like 'Excavator started working' to be more specific.